WindowsFirewallHelper/WindowsFirewallHelper.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2;net4</TargetFrameworks>
+    <TargetFrameworks>netstandard2;net4;net5.0</TargetFrameworks>
```

Comment:
isn't netstandard2 compatible with net5?

Reply:
Assembly System.ServiceProcess.ServiceController, Version=4.2.3.0 not compatible with net5

Reply:
hmm, and neither is v5 with netstandard2?

Reply:
only with netstandard2